dags/bq_events_to_amplitude.py

```diff
@@ -27,7 +27,7 @@
             dag_name=fenix_task_id,
             parent_dag_name=dag_name,
             default_args=default_args,
-            project='moz-fx-data-derived-datasets',
+            project='moz-fx-data-shared-prod',
```

Comment:
`telemetry.fenix_events_v1` is a view published to both projects, so this should have no effect.